Stratis.StratisD/Program.cs

```diff
@@ -35,6 +36,7 @@ public static void Main(string[] args)
                 .UseMempool()
                 .UseWallet()
                 .AddPowPosMining()
+                .UseApi()
```

Comment:
This is required for connection with Breeze. 

But I know what you mean. You are saying that the StratisD project as we distribute it might possibly be just a raw node. But is it the case? 

This builder concept is fine for developers, but how about users? User would rather see some configuration option like `server=1` that would enable the server functionality in the node. But for that you need `.UseServer` in the code (it does not exist, just for the sake of the argument). So it seems to me that there should be some default setting - possibly `api=0` as you suggest, but it should be possible to enable it without rewriting the code. 

So at this point, it is maybe better to have as much functionality enabled in the node to avoid user to have to recompile the node. Later we should add those options that would by default NOT enable those features, but users could use them to enable without recompile.

What do you think?

Reply:
Well the api is enabled in Breeze where it is actually used, I would prefer to keep the daemons here clear of api/rpc for now til we find a proper way to interact with the node in headless mode.

Maybe we we add a cli option to the args, or we expose all rpc via rest.

Reply:
currently i need it just for deving, so i can revert it back if you want, but it probably does not matter at this stage

it makes my life easier now so i added
but if you want to have it removed, i'll remove it in later PR

Reply:
So pls DO let me know if you want this removed